src/main/kotlin/no/nav/familie/ef/sak/ekstern/journalføring/AutomatiskJournalføringService.kt

```diff
@@ -97,4 +127,12 @@ class AutomatiskJournalføringService(
             behandlingService.hentBehandlinger(fagsak.id).none { it.resultat != BehandlingResultat.HENLAGT }
         } ?: true
     }
+
+    private fun utledNesteBehandlingstype(behandlinger: List<Behandling>): BehandlingType {
+        return if (behandlinger.none { it.resultat != BehandlingResultat.HENLAGT }) FØRSTEGANGSBEHANDLING else REVURDERING
+    }
```

Comment:
```suggestion
    private fun utledNesteBehandlingstype(behandlinger: List<Behandling>): BehandlingType {
        return if (behandlinger.all { it.resultat == BehandlingResultat.HENLAGT } || behandlinger.isEmpty()) FØRSTEGANGSBEHANDLING else REVURDERING
    }
```
Jeg måtte stirre litt for å å skjønne dobbel negering 😛  Dette er kanskje lettere å forstå for en random utvikler om 5 år?

Reply:
Må innrømme at du har helt rett her - det er mye enklere å forstå, siden det er det jeg må forklare med ord også 😂

Reply:
@eirarset 👏

Trenger ikke å sjekke `isEmpty`, då all blir true hvis empty :)

Reply:
Ser det, men gir dét egentlig mening?
Det ville ikke jeg antatt i realimplementasjon... 😬